### ISSUE TITLE
Update distinct-attribute documentation is outdated

### ISSUE THREAD

**Issue by ppamorim** [meilisearch/documentation#538]
Hi, the example in the references documentation for the endpoint `distinct-attribute` is outdated and not working anymore on older versions of Meilisearch.

The current value is: `POST - /indexes/:index_uid/settings/distinct-attribute`

The correct value: `POST - http://localhost:7700/indexes/jackets/settings` with `{ "distinctAttribute": "product_id" }` as parameter

Please refer to: https://docs.meilisearch.com/references/distinct_attribute.html#update-distinct-attribute

Regards,
Pedro

**Comment by bidoubiwa** [comment]
It is possible to change the settings in two different route, as with every other setting. Either through the [global `/settings` route](https://docs.meilisearch.com/references/settings.html), or through the specific sub-route, which in this case is [`/settings/distinct_attribute` route](https://docs.meilisearch.com/references/distinct_attribute.html).

Both example you gave should work:


> The current value is: POST - /indexes/:index_uid/settings/distinct-attribute
> The correct value: POST - http://localhost:7700/indexes/jackets/settings with { "distinctAttribute": "product_id" }

**Comment by bidoubiwa** [comment]
My bad, did not realise the sub-route does not exist anymore

**Comment by bidoubiwa** [comment]
My bad, the example in the documentation was not correct it should be 
```bash
  $ curl \
    -X POST 'http://localhost:7700/indexes/movies/settings/distinct-attribute' \
    --data '"movie_id"'
```

fixed by #541
